Riot/Modules/Application/LegacyAppDelegate.m

```diff
@@ -567,6 +571,24 @@ - (void)applicationWillResignActive:(UIApplication *)application
         [wrongBackupVersionAlert dismissViewControllerAnimated:NO completion:nil];
         wrongBackupVersionAlert = nil;
     }
+    
+    if ([self.localAuthenticationService isProtectionSet])
```

Comment:
Is it compatible with the grace period?
It looks that it will display pin protection every time the app will go in bg whatever the grace period.

Reply:
No, i've changed the logic here to avoid content leakage. If the protection set, we're placing a layer (inactive state) on top of window when application resigns active, and when user coming back, we're changing the display to unlock (or destroying it if within the grace period).